CHANGELOG.md

```diff
@@ -9,6 +11,20 @@
   attacher and provisioner into a single Statefulset.
   [[GH-74]](https://github.com/digitalocean/csi-digitalocean/pull/74)
 
+**IMPORTANT**:This release contains breaking changes, mainly about how thing
+are deployed. The minimum Kubernetes version needs to be now **v1.10.5**. 
+To upgrade from a prior `v0.1.x` versions please remove the old CSI plugin
+completely and re-install the new one:
+
+```sh
+# delete old version, i.e: v0.1.5
+kubectl apply -f https://raw.githubusercontent.com/digitalocean/csi-digitalocean/master/deploy/kubernetes/releases/csi-digitalocean-v0.1.5.yaml
```

Comment:
this doesn't delete the old version tho?

Reply:
Oopps, fixed and force pushed